tests/pytests/test_info_modules.py

```diff
@@ -501,6 +502,7 @@ def test_redis_info_modules_vecsim():
   info = env.cmd('INFO', 'MODULES')
   field_infos = [to_dict(env.cmd(debug_cmd(), 'VECSIM_INFO', f'idx{i}', 'vec')) for i in range(1, 4)]
   env.assertEqual(info['search_used_memory_vector_index'], sum(field_info['MEMORY'] for field_info in field_infos))
+
```

Comment:
Consider removing this new line, to avoid annoying conflicts in the future

Reply:
Changes inside the test instead of disabling it entirely to the request of @DvirDukhan, so conflicts are expected either way (also this is temporary for hopefully a few days only...)